lizard_damage/forms.py

```diff
@@ -154,6 +156,14 @@ class FormStep1(forms.Form):
         'behoefte.'
         )
 
+    ahn_version = forms.ChoiceField(
+        label="AHN versie", required=True,
+        choices=(("2", "AHN2"), ("3", "AHN3")),
```

Comment:
Geef je die "2" mee richting database? Dan is "2" een zogenoemd "magisch getal" => maak er een constante van.

In `models.py`:

    AHN2 = "2"
    AHN_VERSIONS = ((AHN2, 'AHNv2'),

En dan die AHN2 hier importeren.

Reply:
In models heb je het AHNv2 genoemd, hier AHN2.